pkg/server/router/credential.go

```diff
@@ -39,33 +39,33 @@ func NewCredentialRouter(s svcframework.Service) (*CredentialRouter, error) {
 
 type CreateCredentialRequest struct {
 	// The issuer id.
-	Issuer string `json:"issuer" validate:"required" example:"did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"`
+	Issuer string `json:"issuer" validate:"required" example:"did:key:z6MkkZDjunoN4gyPMx5TSy7Mfzw22D2RZQZUcx46bii53Ex3"`
 
 	// The KID used to sign the credential.
-	IssuerKID string `json:"issuerKid" validate:"required" example:"#z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"`
+	IssuerKID string `json:"issuerKid" validate:"required" example:"did:key:z6MkkZDjunoN4gyPMx5TSy7Mfzw22D2RZQZUcx46bii53Ex3#z6MkkZDjunoN4gyPMx5TSy7Mfzw22D2RZQZUcx46bii53Ex3"`
```

Comment:
does it work like this even on integ tests? I though we could alwasy do #z6MkkZDjunoN4gyPMx5TSy7Mfzw22D2RZQZUcx46bii53Ex3 and it would work

Reply:
Yes, this is the correct format. Integration tests are picking it up directly from the DID document.